Gruntfile.js

```diff
@@ -8,7 +8,7 @@ module.exports = function (grunt) {
             all: {
                 src: [
                     '*.js',
-                    'bin',
+                    'bin/npm-bump',
```

Comment:
You're right, it's because if you enter a directory here it only matches files with the `.js` extension.